sale_suggested_price/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Coop IT Easy SCRLfs
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "sale_suggested_price",
```

Comment:
Use `Sale Suggested Price` as name. :)

Reply:
@victor-champonnois Did you forget this point or is there a reason for the name of this module with `_` ?